lib/bundler/dsl.rb

```diff
@@ -216,9 +217,9 @@ def group(*args, &blk)
       args.each { @groups.pop }
     end
 
-    def install_if(*args, &blk)
```

Comment:
Please keep the block arg? That way calling without a block will have a better error message